util/rpcserver/rpcserver.go

```diff
@@ -0,0 +1,21 @@
+package rpcserver
+
+import "github.com/spf13/pflag"
+
+type Config struct {
+	Enable        bool `koanf:"enable"`
+	Public        bool `koanf:"public"`
+	Authenticated bool `koanf:"authenticated"`
+}
+
+var DefaultConfig = Config{
+	Enable:        false,
+	Public:        false,
+	Authenticated: true,
+}
+
+func ConfigAddOptions(prefix string, f *pflag.FlagSet) {
+	f.Bool(prefix+".enable", DefaultConfig.Enable, "enable execution node to serve over rpc")
```

Comment:
"enable execution node to serve over rpc" comment should be adjusted by the ConfigAddOptions caller.
How about adding a argument to ConfigAddOptions, nodeType string, and using "enable {nodeType} node to serve over rpc"